docs/releases/1.23-NOTES.md

```diff
@@ -62,6 +62,7 @@ It is recommended to keep using the `v1alpha2` API version.
 
 * IPv6 pod subnets is in a working state using public IPv6 addresses for the Pod network. This works with both Cilium and Calico. IPv6 is still behind a feature flag until service controllers and addons implement support for IPv6. See [the IPv6 documentation](https://kops.sigs.k8s.io/networking/ipv6/).
 
+* The `kops rolling-update cluster` command has a new `--node-drain-timeout` flag for specifying the maximum amount of time to wait when attempting to drain a node.
```

Comment:
I think the change from hanging perpetually to 15 min by default is worth mentioning.